modules/tinymce/src/themes/silver/test/ts/browser/editor/toolbar/MultipleToolbarVisibilityTest.ts

```diff
@@ -0,0 +1,53 @@
+import { Assert, describe, it } from '@ephox/bedrock-client';
+import { Css } from '@ephox/sugar';
+import { McEditor, TinyDom } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+
+describe('browser.tinymce.themes.silver.editor.toolbar.MultipleToolbarVisibilityTest', () => {
```

Comment:
Should this include `Inline` in the test/file name? As this is very specifically an inline mode only issue. Maybe `MultipleInlineToolbarVisibilityTest`?